bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/AbstractComponent.java

```diff
@@ -129,10 +133,8 @@ public boolean isActive() {
      */
     public CompletableFuture<@Nullable Void> start(MqttBrokerConnection connection, ScheduledExecutorService scheduler,
             int timeout) {
-        CompletableFuture<@Nullable Void> all = CompletableFuture.completedFuture(null);
-
-        all = channels.values().stream().map(v -> v.start(connection, scheduler, timeout)).reduce(all,
-                (f, v) -> f.thenCompose(b -> v));
+        CompletableFuture<@Nullable Void> all = channels.values().parallelStream()
```

Comment:
Did you change to parallelStream by intention or is this the result of the auto-complete?

Reply:
Yes. I switched to parallelStream everywhere where we are going into some asynchronious action anyways. Then we can do this in parallel.